javalib/src/main/scala/java/util/Date.scala

```diff
@@ -118,6 +119,8 @@ class Date private (private val date: js.Date) extends Object
       pad0(date.getUTCSeconds().toInt) +" GMT"
   }
 
+  def toInstant(): Instant = Instant.ofEpochMilli(getTime())
```

Comment:
Could you add a (non-Scaladoc) comment on this method mentioning that it is not tested in this repo because it needs `java.time`, please?

And also add the same comment on `from`.

Reply:
Done